src/app/container/container.component.ts

```diff
@@ -197,8 +197,15 @@ export class ContainerComponent extends Entry implements AfterViewInit {
     if (tool) {
       this.tool = tool;
       this.initTool();
+      this.requestAccessHREF$ = this.extendedTool$.pipe(map(data => {
+        if (Object.keys(data).length !== 0 ) {
+          console.log(data);
```

Comment:
Best not to log this.

Reply:
I feel so bad, can't believe I overlooked this...thank you Charles!